docs/guides/rtc_product_guide.md

```diff
@@ -113,7 +113,7 @@ Figure 4 shows the coverage of the various legacy DEM sources.
 
 RTC products can be output at 30-meter, 20-meter, or 10-meter pixel spacing. In most cases, the input SAR image has a resolution closer to the 10-m products, while the Copernicus DEM (used by default for RTC processing) has a pixel spacing of 30 m. The 10-m RTC product will be closer to the resolution of the source SAR granule, but the 30-m RTC product has a much smaller file size. The 20-m product may be a good trade-off between the two.
 
-It is *much* faster to process and analyze 30-m RTC products, so it's a good idea to start with the coarser resolution option if possible. If the 30-m pixel spacing is not sufficient for your use case, try the larger 20-m RTC products. If even more detail is required, the 10-m products may be the best.
+It is *much* faster to process, download and analyze 30-m RTC products than the 10-m products, so it's a good idea to start with the coarser resolution option if possible. If the 30-m pixel spacing is not sufficient for your use case, try the larger 20-m RTC products. If even more detail is required, the 10-m products may be the best option.
```

Comment:
"larger 20-m" might read better as "more-detailed 20-m products", to avoid confusing "larger file size" with "larger pixels"?